index.html

```diff
@@ -259,6 +270,134 @@ <h3>Video Content Hints</h3>
+          <li>For an audio track with the value "speech", and for constraints
+            echoCancellation and autoGainControl, apply a default of
+            "true".</li>
+        </ul>
+      </p>
+      <p>To <dfn>apply a default</dfn> to the constraint <var>c</var>
+        with value <var>t</var>, perform the following steps:
+        <ul>
+          <li>If the value <var>t</var> satisfies the applied constraints,
+            set the setting corresponding to c <var>t</var>.</li>
+          <li>Otherwise, choose a value for the setting corresponding to c
+            that satisfies the applied constraints.</li>
+          <li>Remember the value <var>t</var>.
+          <li>In parallel, update the trac with the new settings.</li>
+        </ul>
+      <p>Whenever the "apply constraints" algorithm is subsequently
+        run, the UA MUST choose the remembered value <var>t</var> if it is
+        now a permitted value.
+      </p>
+      <p>When setting a contentHint value of "", all remembered values of
+        <var>t</var> are removed.
+      </p>
+    </section>
+    <section>
+      <h3>Degradation preference when encoding</h3>
+      <p>
+        When encoding video, and some constraint (bandwidth, CPU) prevents
+        encoding at the configured framerate and resolution, the encoder
+        must make a choice on how to modify the encoding parameters.
+      </p>
+      <p>
+        This section defines terms to describe the choice, and an enum
+        that can be used to indicate that choice in an API.
+      </p>
+      <div>
+        <pre class="idl"
+>enum RTCDegradationPreference {
+  "maintain-framerate",
+  "maintain-resolution",
+  "balanced"
+};</pre>
+        <table data-link-for="RTCDegradationPreference" data-dfn-for=
+        "RTCDegradationPreference" class="simple">
+          <tbody>
+            <tr>
+              <th colspan="2"><code><a>RTCDegradationPreference</a></code> Enumeration description</th>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>maintain-framerate</code></dfn></td>
+              <td>
+                <p>Degrade resolution in order to maintain framerate.</p>
+              </td>
+            </tr>
+            <tr>
+              <td><dfn data-idl><code>maintain-resolution</code></dfn></td>
+              <td>
+                <p>Degrade framerate in order to maintain resolution.</p>
+              </td>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl><code>balanced</code></dfn></td>
+              <td>
+                <p>Degrade a balance of framerate and resolution.</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      An attribute is defined for RTCRtpSendParameters that allows this
+      to be explicitly indicated for an RTCRtpSender:
```

Comment:
s/attribute/member/

Also, we probably need to mention this is settable/getable with `setParameters`/`getParameters`. Specifically, we probably want to move some language here from [getParameters](https://w3c.github.io/webrtc-pc/#dom-rtcrtpsender-getparameters):
 * degradationPreference is set to the last value passed into setParameters, ~or the default value of "balanced" if setParameters hasn't been called.~ or the default value `undefined` if setParameters hasn't been called.

Also, how would one remove an already-set `degradationPreference`? Like this?
```js
const params = sender.getParameters();
delete params.degradationPreference;
await sender.setParameters(params);
```
Does that work, or should we define a literal `"default"` value in the enum?

Reply:
This is a generic question about setParameters, I think. This value shouldn't do anything different to the others.

Reply:
@jan-ivar I believe that would be the way to do it (and that's the way I'm currently implementing it too).